### ISSUE TITLE
p-queue, ironically, requires ESM now

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1553]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.10.7](https://github.com/open-wa/wa-automate-nodejs/compare/3.10.6...3.10.7) (2021-04-04)

- ⬇️  fixes #1553 [`#1553`](https://github.com/open-wa/wa-automate-nodejs/issues/1553)
- Release 3.10.7 [`4e99255`](https://github.com/open-wa/wa-automate-nodejs/commit/4e99255ac56c731375a1eba30485799a188aaff5)
